core/iwasm/compilation/aot_emit_gc.h

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (C) 2019 Intel Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ */
+
+#ifndef _AOT_EMIT_GC_H_
+#define _AOT_EMIT_GC_H_
+
+#include "aot_compiler.h"
+#include "stdbool.h"
```

Comment:
No need to include stdbool.h? It should have been included in platform_internal.h, or bool type should have been defined.